medcat-service/medcat_service/dependencies.py

```diff
@@ -1,26 +1,52 @@
 import logging
-from functools import lru_cache
-from typing import Annotated
+from typing import Annotated, Optional
 
-from fastapi import Depends
+from fastapi import Depends, Request
 
 from medcat_service.config import Settings
 from medcat_service.nlp_processor.medcat_processor import MedCatProcessor
 
 log = logging.getLogger(__name__)
 
+processor_singleton: Optional[MedCatProcessor] = None
+settings_singleton: Optional[Settings] = None
```

Comment:
Is this needed? The relevant methods (`get_global_settings` and `set_global_settings` don't seem to be called anywhere.